### PR TITLE
Follow up for PR #4014 

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Follow up for PR #4014 - Minor language fixes

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@pospispa Can you please review?

**Comment by gaurav-nelson** [comment]
[Document: Add checks and documentation about template pods for recycling](https://trello.com/c/832TMLfa/426-document-add-checks-and-documentation-about-template-pods-for-recycling)

**Comment by pospispa** [comment]
LGTM

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -121,19 +121,24 @@ to policy.
 === Reclaiming
 
 The reclaim policy of a `PersistentVolume` tells the cluster what to do with
-the volume after it is released. Currently, volumes can either be _Retained_, _Recycled_ or _Deleted_.
+the volume after it is released. Volumes reclaim policy can either be `Retained`, `Recycled` or `Deleted`.
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Comma after `Recycled`.

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -121,19 +121,24 @@ to policy.
...
 
-Retention allows for manual reclamation of the resource.  For those volume plugins that support it, deletion removes both the `PersistentVolume` object from Kubernetes and deletes associated storage asset in external infrastructure such as AWS EBS, GCE PD or Cinder volume.  Volumes that were dynamically provisioned are always deleted.
+`Retained` reclaim policy allows manual reclamation of the resource for those volume plugins that support it. `Deleted` reclaim policy deletes both the `PersistentVolume` object from {product-title} and the associated storage asset in external infrastructure, such as AWS EBS, GCE PD or Cinder volume.
```
Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: s/plugins/plug-ins

Comment (bfallonf) [inline architecture/additional_concepts/storage.adoc]: Comma after GCE PD.

**Comment by bfallonf** [comment]
@gaurav-nelson a few nits from me. LGTM otherwise :turtle:

**Comment by ahardin-rh** [comment]
@gaurav-nelson This looks good to merge

**Comment by liangxia** [comment]
Anyone can help merge?

**Comment by gaurav-nelson** [comment]
@liangxia I can merge this, I was waiting for confirmation on the [Trello](https://trello.com/c/832TMLfa/426-document-add-checks-and-documentation-about-template-pods-for-recycling) card

**Comment by gaurav-nelson** [comment]
Language changes only, no rev history needed.

**Pull request merged**